lib/components/RecentBlocks.js

```diff
@@ -42,6 +45,8 @@ class RecentBlocks extends React.Component {
   }
 
   render() {
+    const SPV = getClient().SPV;
```

Comment:
Actually, this makes me think that this should be called `isSPV`. Then the method that actually sets this could be `setSPV` instead? The naming here just doesn't feel quite right.

Reply:
The "setter" is called `isSPV` in bpanel-utils, was just merged:
https://github.com/bpanel-org/bpanel-utils/blob/master/lib/client.js#L231-L241
... but I hear ya. Will update this branch for now and make a new PR for `-utils`

Reply:
Yeah, that was my bad. It only really occurred to me after seeing this in use.

Reply:
updated!